docs/meetings/team/2022_09_27.md

```diff
@@ -0,0 +1,134 @@
+# 🚀 Weekly Team Meeting Minutes
+
+### **Start Time** 9:34pm
+
+### **End Time** 10:30
+
+### **Location** Discord
+
+**Date** Sept 16th 2022
+
+## 👋 Attendence
+
+1. Veronica
+2. Edouard
+3. Matt
+4. Logan
+
+***Attendence Comments***
+
+## 🧸 Check-In
+
+- Dad joke of the day
+
+## Announcements
+
+- Client Meeting needs to be rescheduled/canceled
+- Git Stats due Friday
+- Research before next client meeting
+  
+## Last Week (Restrospective)
+
+### Client meeting
+
+- Important to dicuss with other groups client agenda
+  - Time stamps for each question should become a norm
+  - How much does each person need to get done ?
+- Our group took the meeting over a bit
+- Important to prep questions ahead of time
+  - How do we want to organize the agenda ?
+    - Check in with groups to see how this should get done
+    - Having our own questions would be good and more general ones shared would be okay
+- Languistics, animation, AI/ML, at lot more then orginally thought for the project
+- Thanks matt for bringing the meeting back on track
+
+### Team Assignment
+
+- Exercise went good!
+
+## This Week  
+
+### Full repo set-up
+
+- Send requests to add or change to Logan
+- Naming schemes in repo will follow git standards
+  - Logan to upload docs on this
+  - GitHub Actions to be added
+    - Logan to take a peek at SonarCloud
+- PR template to be added
+  - Logan and Ed can do it
+  
+### Project Management
+
+- Milestones needs to be clarified with the professor
+  - Ask prof during class
+
+### Client meeting Re-schedule
+
+- Matt to make the call later tonight
+  - Potential to have meeting thursday afternoon?
+  - Postpone meeting to monday?
+- Talk with other groups to see whats going on
+- Need to have findings organized before the meeting
+  - Could cancel meeting and then have it next friday
+- Group agrees to postpone meeting and have it next friday
+
+### Research
+
+- Finding related research
+- Don't spend too much reading research, group can gather topics
```

Comment:
only for preliminary research! once we have a good list of resources, we can delve deeper

Reply:
The prelim research should allow us to create exploratory issues that can be broken down into more bite-sized achievable issues